### ISSUE TITLE
How to reduce the requirement of GPU memory

### ISSUE THREAD

**Issue by ghost** [valeoai/ADVENT#4]
Hi, I am trying to reproduce the state-of-the-art of your paper advent.
The server of my school is under system update, so 
For now, I only have one rtx2070s 8GB for training, is there any method to reduce the requirement of GPU memory for training your model?
i have noticed that the batch size is 1 in the config file.

looking forward to your reply! 
many thanks!

**Comment by tuanhungvu** [comment]
Hello @jiaxing002 , I think that you can try training at smaller resolutions. That can be done by changing INPUT_SIZE_. 
T-H

**Comment by soans1994** [comment]
what is the use of batch normalization in your model if bs=1
